__tests__/app.test.js

```diff
@@ -325,3 +325,22 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: Should respond with an array of all user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
If you want to have some fun with destructuring, you can use it in nested fashion:

`.then( ( { body : { users} } ) => { `